### PR TITLE
[batch][ci] tolerate preemptibles

### PR DESCRIPTION
**Pull request by danking**
We've designed them to not have state

### CODE REVIEW

File: batch/deployment.yaml [hunk 1]
```diff
@@ -80,6 +80,11 @@ spec:
          secret:
            optional: false
            secretName: batch-jwt
+       tolerations:
```
Comment (jigold) [inline batch/deployment.yaml]: Should we make this only if we are not in the deploy state?
Example:
```
{% if deploy is sameas false %}
      tolerations:
....
{% endif %}
```

**Comment by jigold** [comment]
Discussed in check-in that this is fine.

**Comment by tpoterba** [comment]
https://ci.hail.is/batches/1420/jobs/32/log

**Comment by johnc1231** [comment]
I think maybe `tolerations` is not currently indented to the same level as `volumes`, causing the error you're getting

**Pull request merged**